### PR TITLE
100 failed upload clean up

### PR DESCRIPTION
**Pull request by norisalsaadie**
Closes #114

### CODE REVIEW

File: src/domain/user/infrastructure/rest-user.ts [hunk 1]
```diff
@@ -13,7 +13,9 @@ export class RestUserRepository implements UserRepository {
             const response = await fetch(this.userAdapterUrl, {
                 headers: [['authorization', header]],
             });
-            return await response.json();
+            const json = await response.json();
+            console.log(json);
```
Comment (diversemix) [inline src/domain/user/infrastructure/rest-user.ts]: Remote the `console.log` ?

File: src/main.ts [hunk 2]
```diff
@@ -118,8 +118,10 @@ const init = async (): Promise<void> => {
         context: ({ req }: ExpressContext): { userId: string; authorizationHeader: string } => {
             try {
                 // @todo: we need to use the correct libero auth token
```
Comment (diversemix) [inline src/main.ts]: Remote the `console.log` ? and L124

File: tests/app.test.ts [hunk 3]
```diff
@@ -163,4 +163,150 @@ describe('Application Integration Tests', () => {
...
+                headers: { Authorization: `Bearer ${jwtToken}` },
+            },
+        );
+
+        expect(deleteResponse.status).toBe(200);
+        console.log('deleteResponse.data.errors', deleteResponse.data.errors);
+        expect(deleteResponse.data.errors).toBeUndefined();
+    });
+
+    it('it should throw a user tries to delete a file unrelated to their submission', async () => {
```
Comment (diversemix) [inline tests/app.test.ts]: Can we have a test for when a user tried to delete a submission that does not belong to them?

File: src/application/wizard/service.ts [hunk 4]
```diff
@@ -52,10 +52,17 @@ export class WizardService {
         return submission;
     }
 
+    async deleteManuscriptFile(fileId: FileId, submissionId: SubmissionId, user: User): Promise<boolean> {
```
Comment (diversemix) [inline src/application/wizard/service.ts]: There are no checks using the permission service - see "saveManuscriptFile"

Comment (norisalsaadie) [reply src/application/wizard/service.ts]: Done and pushed

File: tests/app.test.ts (hunk 3 above)
Comment (norisalsaadie) [reply tests/app.test.ts]: Good point. More testing is needed around this in genera

**Pull request merged**